kubernetes/kserve/developer_guide.md

```diff
@@ -61,36 +61,36 @@ torchserve --start --ts-config config.properties
 Output
 
 ```bash
```

Comment:
This might be a good opportunity to remove these outputs, not sure that it makes things any clearer